lib/kitchen/provisioner/ansible_push.rb

```diff
@@ -39,10 +40,108 @@ class AnsiblePush < Base
       # For tests disable if not needed
       default_config :chef_bootstrap_url, "https://www.getchef.com/chef/install.sh"
 
+      attr_reader :conf
```

Comment:
Syntactic sugar, I can remove it if you prefer; code would look like 

conf()[:some_key]

?